src/ai/backend/client/output/console.py

```diff
@@ -115,41 +117,41 @@ def print_paginated_list(
         self,
         fetch_func: Callable[[int, int], PaginatedResult],
         initial_page_offset: int,
-        page_size: int = None,
+        page_size: Optional[int] = None,
     ) -> None:
-        if sys.stdout.isatty() and page_size is None:
-            page_size = get_preferred_page_size()
-            fields: Sequence[FieldSpec] = []
-
-            def infinite_fetch():
-                nonlocal fields
-                current_offset = initial_page_offset
-                while True:
-                    result = fetch_func(current_offset, page_size)
-                    if result.total_count == 0:
-                        raise NoItems
-                    current_offset += len(result.items)
-                    if not fields:
-                        fields.extend(result.fields)
-                    yield from result.items
-                    if current_offset >= result.total_count:
-                        break
+        fields: List[FieldSpec] = []
+
+        def infinite_fetch(_page_size) -> Iterator[_Item]:
```

Comment:
Please add type annotation to `_page_size` parameter also.
```suggestion
        def infinite_fetch(_page_size: int) -> Iterator[_Item]:
```

Reply:
Oh, I missed it. Thank you.